composer.json

```diff
@@ -30,7 +30,7 @@
     },
     "require-dev": {
         "cakephp/cakephp-codesniffer": "~2.1",
-        "phpunit/phpunit": "5.*"
+        "phpunit/phpunit": "<6.0"
```

Comment:
What's the point ? I don't understand as composer will always fetch the latest version possible and debug_kit is php 5.5...